packages/search-ui-workplace-search-connector/src/WorkplaceSearchAPIConnector.ts

```diff
@@ -224,33 +274,17 @@ class WorkplaceSearchAPIConnector {
     };
 
     return this.beforeSearchCall(options, async (newOptions) => {
-      const searchResponse = await fetch(
-        `${this.enterpriseSearchBase}/api/ws/v1/search`,
-        {
-          method: "POST",
-          headers: {
-            "Content-Type": "application/json",
-            Authorization: `Bearer ${this.accessToken}`
-          },
-          body: JSON.stringify({
-            query,
-            ...newOptions
-          })
-        }
-      );
+      const apiUrl = `${this.enterpriseSearchBase}/api/ws/v1/search`;
 
-      if (searchResponse.status === 401) {
-        this.state.isLoggedIn = false; // Remove the token to trigger the Log in dialog
-        throw new Error(INVALID_CREDENTIALS);
-      }
-
-      const responseJson = await searchResponse.json();
-
-      // const response = await this.client.search(query, newOptions);
+      const responseJson = await this.performFetchRequest(apiUrl, {
+        query,
+        ...newOptions
+      });
       return adaptResponse(
         responseJson,
         buildResponseAdapterOptions(queryConfig)
       );
+      // const response = await this.client.search(query, newOptions);
```

Comment:
I think we should remove these uncommented lines as likely dont add value keeping them here.

Reply:
Yeah, nice catch, missed it.